<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TWW HD RPX Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>515</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open A &quot;The Wind Waker HD&quot; RPX File To Start. </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-25</y>
      <width>515</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>188</x>
        <y>50</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Boomerang Trail</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <item>
       <property name="text">
        <string>Boomerang Trail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sword Swing Trail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sword Parry Trail</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sword Soup Trail</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>203</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Color Modification</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>130</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Color</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>405</x>
        <y>200</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="statusTip">
       <string>Removes the sail when traveling in the sea if checked.</string>
      </property>
      <property name="text">
       <string>Remove Sail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>200</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>180</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Transparency</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Special thanks to SuperDude88YT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentText">
       <string>Forsaken Fortress</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Forsaken Fortress</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Star Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Northern Fairy Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Gale Isle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Crescent Moon Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Seven-Star Isles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Overlook Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Four-Eye Reef</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mother and Child Isles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectacle Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Windfall Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Pawprint Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dragon Roost Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Flight Control Platform</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Western Fairy Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rock Spire Isle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tingle Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle Part One</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eastern Fairy Island</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fire Mountain</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Star Belt Archipelago</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Three-Eye Reef</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Greatfish Isle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cyclops Reef</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Six-Eye Reef</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Outset Island</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>130</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Island</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>180</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Spawn Point</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>100</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>New Save Starting Island</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>99</y>
        <width>181</width>
        <height>131</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_6</zorder>
     <zorder>label_3</zorder>
     <zorder>checkBox</zorder>
     <zorder>label</zorder>
     <zorder>spinBox</zorder>
     <zorder>label_5</zorder>
     <zorder>label_7</zorder>
     <zorder>comboBox_2</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>comboBox_3</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>515</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Press Ctrl+O or File -&gt; Open to select the RPX.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>tabWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>515</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionDefault"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDefault">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
